includes/class.llms.post-types.php

```diff
@@ -203,10 +241,10 @@ public static function get_post_type_caps( $post_type ) {
 	/**
 	 * Retrieve taxonomy capabilities for custom taxonomies
```

Comment:
```suggestion
	 * Retrieve taxonomy capabilities for custom taxonomies.
```

Reply:
The "sumary" acts as a title does not require a period. This is per our docs and the wpcs docs. This isn't getting flagged by phpcs not because i disable the rule but because it's not a rule we're following. I do admit that sometimes I accidentally add a period and when I see a period I don't worry about it because it's not an offense but it is NOT required.

https://github.com/gocodebox/lifterlms/blob/master/docs/documentation-standards.md#summary

And now reading the docs I see the example has a period so that's a bit misleading.

Sorry.